src/Microsoft.AspNet.Hosting/Internal/HostingEngine.cs

```diff
@@ -87,6 +88,7 @@ public virtual IDisposable Start()
                 _applicationLifetime.NotifyStopping();
                 server.Dispose();
                 _applicationLifetime.NotifyStopped();
+                loggerFactory.Dispose();
```

Comment:
No, Hosting depends on Logging. That Logging PR is nice to have, but calling Dispose not a blocking issue for beta7.

Your partial fix here does not contribute towards the full fix, so don't bother.